src/components/Data/Documents/Views/Column.vue

```diff
@@ -313,6 +324,39 @@ export default {
     }
   },
   methods: {
+    isObject: _.isObject,
+    promptExportCSV() {
+      this.$bvModal
```

Comment:
Nitpicking: you can `await` the confirmation modal instead of attaching a `then` callback